code/game/jobs/job/antag/xeno/queen.dm

```diff
@@ -19,5 +19,7 @@
 	to_chat(new_queen, "Talk in Hivemind using <strong>;</strong> (e.g. ';Hello my children!')")
 
 AddTimelock(/datum/job/antag/xenos/queen, list(
-	JOB_XENO_ROLES = 10 HOURS
+	JOB_XENO_ROLES = 10 HOURS,
+	JOB_DRONE_ROLES = 10 HOURS,
```

Comment:
Over 7+ Rounds as a Drone? Seems a bit overkill, lower to 5 Hours as drone and even that is long, and 3h of t3.

This would still be one of the highest requirement for a role if not the highest. Plus people will just afk if this is such a long timer and lack of queen players is already a thing on lowpop.

Reply:
People can evolve without timelock which is what happens most of the time on lowpop anyways..